.github/actions/deploy/action.yml

```diff
@@ -110,43 +103,33 @@ runs:
         TF_VAR_azure_credentials: ${{ inputs.azure-credentials }}
         CONFIRM_PRODUCTION: true
 
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true
+
+    - uses: DFE-Digital/keyvault-yaml-secret@v1
+      id: get_smoke_test_secrets
+      with:
+        keyvault: ${{ env.KEY_VAULT_NAME }}
+        secret: APPLY-CYPRESS-SECRETS
+        key: CANDIDATE_TEST_EMAIL
+
+    - name: Run smoke tests
+      if: env.DEPLOY_ENV == 'review-7345'
```

Comment:
That particular condition can be removed now, but it's a good question whether it should be replaced by something.  I think it can run against everything if you like.

Reply:
yh i think "run against everything" is probably the right place to start.